.devcontainer/devcontainer.json

```diff
@@ -1,18 +1,29 @@
 {
 	"name": "Python 3.6 & PostgreSQL",
-	"dockerComposeFile": "../docker-compose.yml",
-	"service": "app",
+	"dockerComposeFile": "docker-compose.yml",
+	"service": "dev",
 	"workspaceFolder": "/workspace",
 
 	"settings": { 
-		"terminal.integrated.shell.linux": "/bin/sh",
-		"python.pythonPath": "/usr/local/bin/python",
+		"editor.fontFamily": "FiraCode Nerd Font",
 		"python.linting.enabled": true,
 		"python.linting.pylintEnabled": true,
-		"python.linting.pylintPath": "/usr/local/bin/pylint"
+		"python.linting.pylintPath": "/usr/local/bin/pylint",
+		"python.pythonPath": "/usr/local/bin/python",
+		"terminal.integrated.shell.linux": "/bin/bash",
+		"terminal.integrated.fontFamily": "FiraCode Nerd Font Mono"
 	},
 
 	"extensions": [
-		"ms-python.python"
+		"donjayamanne.python-extension-pack",
+		"ms-azuretools.vscode-docker",
+		"ms-python.vscode-pylance",
+		"eamodio.gitlens",
+		"wholroyd.jinja",
+		"pmbenjamin.vscode-snyk",
+		"visualstudioexptteam.vscodeintellicode",
+		"yzhang.markdown-all-in-one",
+		"ms-ossdata.vscode-postgresql",
+		"googlecloudtools.cloudcode"
 	]
```

Comment:
Sounds like personal preferences to me. Maybe not the extensions, but definitely the font 😉  Can we leave some things out?

Reply:
Yeah, some extensions in there are quite standard for development and Python. I am conflicted with the font because the default font provided by the integrated console lacks many characters that modern prompt makes use of, but __I can certainly remove the font for the vscode editor at large__. Although, I must say nerd font is pretty nice and popular among devs in general.